<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CommunicationClass</class>
 <widget class="QMainWindow" name="CommunicationClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1090</width>
    <height>597</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>通信模块</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>201</width>
      <height>511</height>
     </rect>
    </property>
    <property name="title">
     <string>串口设置</string>
    </property>
    <widget class="QPushButton" name="serial_switch">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>打开串口</string>
     </property>
    </widget>
    <widget class="QPushButton" name="serialConfigButton">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>串口配置</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>10</y>
      <width>201</width>
      <height>271</height>
     </rect>
    </property>
    <property name="title">
     <string>网口设置</string>
    </property>
    <widget class="QPushButton" name="pushButtonConnect">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>打开服务器</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonDisConnect">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>210</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>关闭服务器</string>
     </property>
    </widget>
    <widget class="QPushButton" name="tcpConfigButton">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>120</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Tcp配置</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="serial_display_group">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>331</width>
      <height>511</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="send_button">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>470</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>发送</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="log_display">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>311</width>
       <height>321</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="tx_data_clear">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>470</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>清除发送</string>
     </property>
    </widget>
    <widget class="QPushButton" name="log_clear">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>360</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>清除log</string>
     </property>
    </widget>
    <widget class="QPushButton" name="rxtx_clear">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>360</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>清除计数</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="tx_display">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>360</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="send_data_display">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>400</y>
       <width>311</width>
       <height>61</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="rx_display">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>360</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="tcp_display_group">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>10</y>
      <width>331</width>
      <height>511</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="pushButtonSend">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>470</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>发送</string>
     </property>
    </widget>
    <widget class="QPushButton" name="log_clear_tcp">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>360</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>清除log</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="tx_display_tcp">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>360</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="rxtx_clear_tcp">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>360</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>清除计数</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEditLog">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>311</width>
       <height>321</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="rx_display_tcp">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>360</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="tx_data_clear_tcp">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>470</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>清除发送</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditTextToSend">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>400</y>
       <width>311</width>
       <height>61</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1090</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
